### ISSUE TITLE
Sample code compatibility with Keycloak v26+?

### ISSUE THREAD

**Issue by bicpi** [PacktPublishing/Keycloak---Identity-and-Access-Management-for-Modern-Applications-2nd-Edition#23]
Hi, I'm following the code examples in the book but things might have changed in newer Keycloak version? I am using the latest version 26.

The frontend from chapter 2 starts but shows a white screen. From the console output I can see that the `index.html` cannot instantiate the `Keycloak` object because the class does not exist.

I guess the class should come from `KC_URL/js/keycloak.js` file loaded in the HTML head of the `index.html` but this file does not exist in Keycloak which gives a 404 there.

Do you may have a hint or updated examples to get this working?
Thanks a lot!

**Comment by jose1003** [comment]
With this version is working: docker run -e KEYCLOAK_ADMIN=admin -e KEYCLOAK_ADMIN_PASSWORD=admin -p 8080:8080 quay.io/keycloak/keycloak:22.0.0 start-dev
, I am trying to find where is the file in the latest version of keycloak

**Comment by bicpi** [comment]
Yes, thanks, I used version 22 then to follow the examples in the book and it worked fine :-)

An idea might be to create a branch for a later Keycloak version(s) when the there are significant changes.

Thanks!
